src/Agoda.Analyzers/AgodaCustom/AG0039MethodLineLengthAnalyzer.cs

```diff
@@ -31,37 +40,29 @@ public class AG0039UndocumentedMemberAnalyzer : DiagnosticAnalyzer
             Category,
             DiagnosticSeverity.Warning,
             isEnabledByDefault: true,
-            description: "Public surfaces should be documented, please add XML documentation.");
-
-        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(Descriptor);
+            description: "Method is longer than 120 lines of code, please consider refactoring to make it more readable.");
```

Comment:
Should this have a substitution wtih the `MaxLines` constant? I think the number in the text is out of sync with the const

Reply:
have removed it from the error